code/game/objects/machinery/doors/airlock.dm

```diff
@@ -203,6 +229,31 @@
 			. += image(icon, "panel_open")
 		if(welded)
 			. += image(icon, "welded")
+	if(CHECK_BITFIELD(machine_stat, PANEL_OPEN) || welded)
+		overlays = list()
```

Comment:
don't do this. Look at above, or at the parent of update_overlays.
. is a list, all overlays are added to . and they are added after the fact.